defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 ntp_enabled: true
-ntp_timezone: America/Chicago
+ntp_timezone: Etc/UTC
```

Comment:
Can you split this change into a separate PR? I can easily incorporate that right away, but I want to test the other changes more thoroughly.